src/subscriber/subscriberV2.ts

```diff
@@ -1,4 +1,6 @@
+import util from 'util';
 import { SubscriberOptions as GoogleCloudSubscriberOptions } from '@google-cloud/pubsub/build/src/subscriber';
+import { Logger } from '../service/logger';
```

Comment:
Codacy has a fix for the issue: [Replace `{·Logger·}·from·'../service/logger` with `util·from·'util`](https://app.codacy.com/gh/deliveryhero/hfc-pubsub/pullRequest?prid=8095345)
```suggestion
import util from 'util';
```